pkgs/applications/audio/yasr/30_conf

```diff
@@ -0,0 +1,22 @@
+Index: yasr/yasr.conf
+===================================================================
+--- yasr.orig/yasr.conf	2008-01-31 14:01:50.000000000 +0100
++++ yasr/yasr.conf	2012-01-28 21:43:09.027710410 +0100
+@@ -55,13 +55,13 @@ cursor tracking=full
+ #synthesizer=doubletalk
+ #synthesizer=bns
+ #synthesizer=apollo
+-synthesizer=emacspeak server
+-#synthesizer=speech dispatcher
++#synthesizer=emacspeak server
++synthesizer=speech dispatcher
```

Comment:
does these configuration changes also make sense for Nixos? Is `speech` to be preferred over `emacspeak`?
 At least the `shell=/bin/bash` should be `shell=/bin/sh` (sh is symlink to bash)